src/Microsoft.AspNet.Mvc.ViewFeatures/HtmlTextWriter.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNet.Html.Abstractions;
+
+namespace Microsoft.AspNet.Mvc.Razor
```

Comment:
Think `Microsoft.AspNet.Mvc.Rendering` would be much more appropriate. This class is not doing anything Razor-related.